st2actions/st2actions/runners/python_action_wrapper.py

```diff
@@ -81,6 +88,10 @@ def _get_action_instance(self):
         config_parser = ContentPackConfigParser(pack_name=self._pack)
         config = config_parser.get_action_config(action_file_path=self._file_path)
 
+        action_cls.datastore = DatastoreService(logger=self._set_up_logger(),
```

Comment:
You're right, I intended to set it on the instance, not that class. I will correct it.